metadata-ingestion/src/datahub/cli/ingest_cli.py

```diff
@@ -72,6 +72,13 @@ def ingest() -> None:
     default=False,
     help="If enabled, ingestion runs with warnings will yield a non-zero error code",
 )
+@click.option(
+    "--safe",
+    type=bool,
+    is_flag=True,
+    default=False,
+    help="Supress display of credentials in logs by supressing elaborae stacktrace (stackprinter) during ingestion failures",
```

Comment:
Ditto. We should say "Suppress display of error logs during ingestion failures"

Reply:
Done. Thanks!